packages/cxl-ui/scss/utils/_mq.scss

```diff
@@ -0,0 +1,34 @@
+// To enable support for browsers that do not support @media queries,
+// (IE <= 8, Firefox <= 3, Opera <= 9) set $mq-responsive to false
+// Create a separate stylesheet served exclusively to these browsers,
+// meaning @media queries will be rasterized, relying on the cascade itself
+$mq-responsive: true;
+
+// Name your breakpoints in a way that creates a ubiquitous language
+// across team members. It will improve communication between
+// stakeholders, designers, developers, and testers.
+$mq-breakpoints: (
+  mobile: 320px,
+  tablet: 740px,
+  desktop: 980px,
+  wide: 1300px,
+  /* Tweakpoints*/ desktopad: 810px,
+  mobilelandscape: 480px
+);
+
+// Define the breakpoint from the $mq-breakpoints list that should
+// be used as the target width when outputting a static stylesheet
+// (when $mq-responsive is set to 'false').
+$mq-static-breakpoint: desktop;
+
+// If you want to display the currently active breakpoint in the top
+// right corner of your site during development, add the breakpoints
+// to this list, ordered by width. For example: (mobile, tablet, desktop).
+$mq-show-breakpoints: (mobile, mobilelandscape, tablet, desktop, wide);
+
+// Wrap width
+$wrap_desktop: 960px;
+$wrap_wide: 1152px;
```

Comment:
Wouldn't this be some kind of formula?

`calc( $mq-breakpoints['mobile'] - ( var(--lumo-sizing-xl * 2 ) )` or similar?

Reply:
I also don't understand why we need mq-specific variable values at all.